router_test.go

```diff
@@ -227,7 +227,7 @@ func TestWildcardBeforeEncodingProcedureRegistration(t *testing.T) {
 		"expected router panic")
 }
 
-func IgnoreTestRouterWithMiddleware(t *testing.T) {
+func TestIgnoreTestRouterWithMiddleware(t *testing.T) {
```

Comment:
Should this be `TestIgnoreRouterWithMiddleware` now?

Reply:
🤦‍♂️  probably